samples/sp-webpart/src/webparts/mgtDemo/MgtDemoWebPart.ts

```diff
@@ -3,6 +3,7 @@ import * as ReactDom from 'react-dom';
 import { Version } from '@microsoft/sp-core-library';
 import { BaseClientSideWebPart } from '@microsoft/sp-webpart-base';
 import { IPropertyPaneConfiguration, PropertyPaneTextField } from '@microsoft/sp-property-pane';
+import { lazyLoadComponent } from '@microsoft/mgt-spfx-utils/dist/es6/lazyLoadMgtReactComponent';
```

Comment:
/dist/es6?

Reply:
I could add a barrel type to expose the helper functions at the root of the package

Reply:
Watch out with barrels as they can lead to circular references. When you load one thing from the barrel, you load everything from it, so the convenience of readability might not be worth the performance hit.